src/seedu/addressbook/AddressBook.java

```diff
@@ -564,6 +568,8 @@ private static String getMessageForSuccessfulDelete(String[] deletedPerson) {
      * @return feedback display message for the operation result
      */
     private static String executeClearAddressBook() {
+        if(!requestConfirmation(COMMAND_CLEAR_DESC))
```

Comment:
According to the [style guide](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used), you always need to have braces, even if it is a single line.

And there should always be a space between the `if` and the opening parenthesis.